operations/helm/charts/mimir-distributed/templates/podsecuritypolicy.yaml

```diff
@@ -6,33 +6,36 @@ metadata:
   labels:
     {{- include "mimir.labels" (dict "ctx" .) | nindent 4 }}
   annotations:
-    "seccomp.security.alpha.kubernetes.io/allowedProfileNames": runtime/default
+    "seccomp.security.alpha.kubernetes.io/allowedProfileNames": {{ .Values.rbac.podSecurityPolicy.seccompProfile }}
 spec:
-  privileged: false
-  allowPrivilegeEscalation: false
+  privileged: {{ .Values.rbac.podSecurityPolicy.privileged }}
+  allowPrivilegeEscalation: {{ .Values.rbac.podSecurityPolicy.allowPrivilegeEscalation }}
   volumes:
     - 'configMap'
     - 'emptyDir'
     - 'persistentVolumeClaim'
     - 'secret'
-  hostNetwork: false
-  hostIPC: false
-  hostPID: false
+    {{- range $volumes := .Values.rbac.podSecurityPolicy.additionalVolumes }}
+    - '{{ $volumes }}'
+    {{- end }}
+  hostNetwork: {{ .Values.rbac.podSecurityPolicy.hostNetwork }}
+  hostIPC: {{ .Values.rbac.podSecurityPolicy.hostIPC }}
+  hostPID: {{ .Values.rbac.podSecurityPolicy.hostPID }}
   runAsUser:
-    rule: 'MustRunAsNonRoot'
+    rule: {{ .Values.rbac.podSecurityPolicy.runAsUser.rule }}
   seLinux:
-    rule: 'RunAsAny'
+    rule: {{ .Values.rbac.podSecurityPolicy.seLinux.rule }}
   supplementalGroups:
-    rule: 'MustRunAs'
+    rule: {{ .Values.rbac.podSecurityPolicy.supplementalGroups.rule }}
     ranges:
-    - min: 1
-      max: 65535
+    - min: {{ .Values.rbac.podSecurityPolicy.supplementalGroups.min }}
+      max: {{ .Values.rbac.podSecurityPolicy.supplementalGroups.max }}
   fsGroup:
-    rule: 'MustRunAs'
+    rule: {{ .Values.rbac.podSecurityPolicy.fsGroup.rule }}
     ranges:
-    - min: 1
-      max: 65535
-  readOnlyRootFilesystem: true
+    - min: {{ .Values.rbac.podSecurityPolicy.fsGroup.min }}
+      max: {{ .Values.rbac.podSecurityPolicy.fsGroup.max }}
```

Comment:
the templates for min and max don't use the list from the values. You can see that the diff in this file doesn't render properly

operations/helm/tests/enterprise-https-values-generated/mimir-distributed/templates/podsecuritypolicy.yaml

Reply:
I updated the template and it works now :)